### PR TITLE
Update the command for disabling survey

### PR DESCRIPTION
**Pull request by evelyn-ys**
It should be `core.survey_message`, not `core.display_survey_message`

### CODE REVIEW

File: docs-ref-conceptual/command-line-tools-survey-guidance.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ command or via an environment variable.
 In the following example, the `az config` command is used to disable the survey message.
 
 ```azurecli
-az config set core.display_survey_message=false
+az config set core.survey_message=false
 ```
 
 You can also use the `AZSURVEYMESSAGE` environment variable to disable the survey message.
```
Comment (evelyn-ys) [inline docs-ref-conceptual/command-line-tools-survey-guidance.md:38]:
```suggestion
You can also use the `AZURE_SURVEY_MESSAGE` environment variable to disable the survey message.
```

**Review by jsntcy — APPROVED** [review]

File: docs-ref-conceptual/command-line-tools-survey-guidance.md [hunk 2]
```diff
@@ -32,10 +32,10 @@ command or via an environment variable.
 In the following example, the `az config` command is used to disable the survey message.
 
 ```azurecli
-az config set core.display_survey_message=false
+az config set core.survey_message=false
 ```
 
-You can also use the `AZSURVEYMESSAGE` environment variable to disable the survey message.
+You can also use the `AZURE_SURVEY_MESSAGE` environment variable to disable the survey message.
```
Comment (evelyn-ys) [inline docs-ref-conceptual/command-line-tools-survey-guidance.md:38]:
```suggestion
You can also use the `AZURE_CORE_SURVEY_MESSAGE` environment variable to disable the survey message.
```

**Review by yonzhan — APPROVED** [review]

**Comment by dnelson-relativity** [comment]
@evelyn-ys Do you mind providing some clarity on using the environment variable? Would it be one of these?
```
AZURE_CORE_SURVEY_MESSAGE=false
```
```
AZURE_CORE_SURVEY_MESSAGE=0
```

**Comment by evelyn-ys** [comment]
@dnelson-relativity The first one is the correct setting.
```
AZURE_CORE_SURVEY_MESSAGE=false
```

**Pull request merged**